aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -210,6 +210,9 @@ MPSDataType getMPSScalarType(ScalarType scalar_type) {
       return "float";
     case ScalarType::Half:
       return "half";
+    case ScalarType::BFloat16:
+      checkSupportsBFloat16();
```

Comment:
I guess this raises a nice error message that this dtype is only support for 14+?

Reply:
Yes
https://github.com/pytorch/pytorch/blob/a6149eba1268d240c30c9fc220f2fa0af8b5cc0c/aten/src/ATen/native/mps/OperationUtils.mm#L58